src/Docker/Image/AWSElasticContainerRegistry.php

```diff
@@ -44,12 +46,24 @@ public function getAwsAccountId()
      */
     public function getLoginParams()
     {
+        $stsClient = new StsClient([
+            'region' => $this->getAwsRegion(),
+            'retries' => self::CONNECT_RETRIES,
+            'connect_timeout' => self::CONNECT_TIMEOUT,
+            'timeout' => self::TRANSFER_TIMEOUT,
+            'credentials' => false,
```

Comment:
Pro volání `assumeRoleWithWebIdentity` musí být na `false` jde o call který se neautentizuje AWS sig v4 ale OIDC tokenem. Jineho providera ktery pouziva STS ale nepouzivame - bud to pouzije ENV vars (azure, testy) nebo web identity (OIDC).